.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
+# . "$(dirname "$0")/_/husky.sh"
```

Comment:
just don't forget to uncommit this

you can run with `git commit --no-verify` if you need to bypass the formatting

Reply:
Yep, thanks for pointing this out - had to hack it when was doing a big rebase via Goland UI